src/github.com/getlantern/lantern-android/client/client.go

```diff
@@ -52,32 +54,11 @@ func NewClient(addr, appName string) *MobileClient {
 
 	hqfd := client.Configure(clientConfig.Client)
 
-	hqfdc := hqfd.NewDirectDomainFronter()
-
-	// store GA session event
-	sessionPayload := &analytics.Payload{
-		HitType: analytics.EventType,
-		Event: &analytics.Event{
-			Category: "Session",
-			Action:   "Start",
-			Label:    runtime.GOOS,
-		},
-	}
-
-	// attach our app-specific analytics tracking code
-	// to session info
-	if appName != "" {
-		if trackingId, ok := trackingCodes[appName]; ok {
-			sessionPayload.TrackingId = trackingId
-		}
-	}
-
-	analytics.SessionEvent(hqfdc, sessionPayload)
-
 	return &MobileClient{
 		Client:  client,
 		closed:  make(chan bool),
-		fronter: hqfdc,
+		fronter: hqfd.NewDirectDomainFronter(),
```

Comment:
We could maintain this locally in `pollConfiguration`, since it's only ever used by its callees.  But it's not a big deal to have it here either.